### ISSUE TITLE
Null Object in the model when an index is modified

### ISSUE THREAD

**Issue by lmouline** [kevoree-modeling/framework#19]
A simple metamodel:

```
class kmf.test.Personne {
    att name : String with index
    att birthMonth : String
}
```

And a simple code where a create A person with a name "Paul". Then, in a next time, I change his name to "Marco". [I attach the main method to this issue](https://github.com/kevoree-modeling/framework/files/49152/TestIndex.txt) (in a TXT, thanks GitHUb XD)

This produces the following output:

```
The model on (0,0) is: 
        [{"universe":0,"time":0,"uuid":1,"data":{"birthMonth":"April","name":"Paul"}}]
The model on (0,1) is: 
        [null, {"universe":0,"time":1,"uuid":1,"data":{"birthMonth":"April","name":"Marco"}}]
```

**Comment by dukeboard** [comment]
By model you mean in the FindAll result ? (namely in the index ?)

**Comment by lmouline** [comment]
yes

**Comment by dukeboard** [comment]
Fixed in the trunk, will be released in the V27.0 version branch
